core/runtime/mod.rs

```diff
@@ -1,5 +1,5 @@
 // Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
-mod bindings;
+pub(crate) mod bindings;
```

Comment:
Why does it need to public for a crate now?

Reply:
There's a utility function in `bindings` required for ModuleMap 😢 

```
pub fn create_empty_fn<'s>(
  scope: &mut v8::HandleScope<'s>,
) -> Option<v8::Local<'s, v8::Function>> {
  let empty_fn = v8::FunctionTemplate::new(scope, empty_fn);
  empty_fn.get_function(scope)
}
```

Reply:
I could probably just duplicate it as it's not really that complex

Reply:
Actually, this requires an external so it does need to be pub(crate) for now. I can clean this up in a later refactoring but for now it's too painful.

Reply:
I agree, seems better to duplicate that export that module.